manifests/kustomize/third-party/tekton-custom-task/pipeline-loops/300-pipelineloop.yaml

```diff
@@ -10,29 +10,29 @@ metadata:
 spec:
   group: custom.tekton.dev
   preserveUnknownFields: false
```

Comment:
>In apiextensions.k8s.io/v1 pruning will be the default, with ways to opt-out of it. Pruning in apiextensions.k8s.io/v1beta1 is enabled via 
 `preserveUnknownFields: false` 

This is a default behaviour in v1. [link: ](https://kubernetes.io/blog/2019/06/20/crd-structural-schema/#pruning-don-t-preserve-unknown-fields)

Reply:
thanks for pointing this out, I removed it from the pipelineloop crd.